DataObjects/DataObjects/GeantEventEntry.h

```diff
@@ -0,0 +1,23 @@
+#ifndef DATAOBJECTS_GEANTEVENTENTRY_H
+#define DATAOBJECTS_GEANTEVENTENTRY_H
+
+//Include files
+#include "GaudiKernel/DataObject.h"
+#include "G4Event.hh"
+
+class GAUDI_API GeantEventEntry : public DataObject {
+ public:
+  GeantEventEntry() : m_event(0){};
+   virtual ~GeantEventEntry() {if (m_event!=0) delete m_event;};
```

Comment:
also, this should probably be moved to a `.cxx` file otherwise there won't be any anchor for the vtable (and this will bloat the executable)